src/main/java/frc/robot/Robot.java

```diff
@@ -611,9 +615,43 @@ private void executeShootingStateMachine() {
   }
 
   private void executeClimbingStateMachine() {
-    switch (mClimingState) {
+    switch (mClimbingState) {
       case IDLE:
+        mRobotLogger.warn("Climbing state is idle");
```

Comment:
Idle state should stop the climber motor.

Reply:
Fixed